README.md

```diff
@@ -3,116 +3,104 @@ Universal neural network aimbot for all games with custom training mode
 
 ## NVIDIA ONLY
 
-This cheat on machine learning, which recognizes objects in a certain range, then leads to an object and makes a shot. He does not interfere in any way with the memory of the game, he simply takes screenshots and recognizes objects.
+This cheat uses machine learning, which recognizes objects in a certain range, then aims at the object and shoots. It does not interfere in any way with the memory of the game, it simply takes screenshots and recognizes objects.
 Of course, this is not the final version, in the future we will have to wait for improvements of a different type (amd video card support (if I have one for the test :)), smooth aim, AdvancedRCS for any games, improvements in object recognition, etc.).
 
-**To launch sources download release version and copy file to DEBUG folder!**
+**Training dependencies must be downloaded from the release version and copied to the Debug folder!**
 
 **Upload trained files of games in the "Trained files for games" folder via pull requests!**
 
-**Or if u can't then write me in telegram: CowDrow or Discord: ZxCL#1220!**
+**If you can't upload, then write to me in Telegram: CowDrow, or Discord: ZxCL#1220!**
```

Comment:
If you can not upload the files, then send me a message on Discord: ZxCL#1220 or Telegram: CowDrow.